cmd/net.go

```diff
@@ -198,13 +198,15 @@ func extractHostPort(hostAddr string) (string, string, error) {
 		return "", "", errors.New("unable to process empty address")
 	}
 
+	// Simplify the work of url.Parse() and always send a url with
+	if !strings.HasPrefix(hostAddr, "http://") && !strings.HasPrefix(hostAddr, "https://") {
+		hostAddr = "//" + hostAddr
+	}
+
 	// Parse address to extract host and scheme field
 	u, err := url.Parse(hostAddr)
 	if err != nil {
-		// Ignore scheme not present error
-		if !strings.Contains(err.Error(), "missing protocol scheme") {
```

Comment:
> This error message check is still valid.

Actually, we were *ignoring* `missing protocol scheme` but now we won't ignore it just because it will never happen in practice.